opal-gwt-client/src/main/java/org/obiba/opal/web/gwt/app/client/i18n/Translations.java

```diff
@@ -610,6 +610,7 @@
       "ProjectNameMustBeUnique", "A project already exists with this name.",//
       "ProjectCreationFailed", "The datasource creation has failed: {0}.", //
       "FileNotFound", "File not found: {0}.",//
+      "InvalidViewBackupFile", "A backup file does not contain a valid ViewDto.",
```

Comment:
ViewDto is internal wording, change for "The backup file does not contain a valid view (JSON format expected)"